src/lib/components/jsonEditorComponents/index.js

```diff
@@ -0,0 +1,10 @@
+export * from "./styles";
```

Comment:
this is unnecessary, remove this and make sure that imports can be imported as
```
import Boolean from './Boolean'
import ParentLabel from './ParentLabel'
```

Reply:
Resolved by using direct imports